CHANGELOG.md

```diff
@@ -67,6 +67,7 @@
 * [FEATURE] QueryFrontend: Support vertical sharding for subqueries. #4955
 * [FEATURE] Querier: Added a new limit `-querier.max-fetched-data-bytes-per-query` allowing to limit the maximum size of all data in bytes that a query can fetch from each ingester and storage. #4854
 * [FEATURE] Added 2 flags `-alertmanager.alertmanager-client.grpc-compression` and `-querier.store-gateway-client.grpc-compression` to configure compression methods for grpc clients. #4889
+* [FEATURE] Enable experimental out-of-order samples support. Added 2 new configs `out_of_order_time_window` and `out_of_order_cap_max`. Removed config `reject_old_samples` and `reject_old_samples_max_age`. #4964
```

Comment:
This changelog needs updating, since we are not revmoing reject-* anymore :)

Reply:
Updated